mlir/test/Target/LLVMIR/llvmir-intrinsics.mlir

```diff
@@ -712,6 +712,13 @@ llvm.func @coro_resume(%arg0: !llvm.ptr) {
   llvm.return
 }
 
+// CHECK-LABEL: @coro_promise
```

Comment:
Can you add a check that matches the intrinsics declaration at the end of the file, e.g., right after:
`// CHECK-DAG: declare void @llvm.coro.resume(ptr)`

Reply:
For some reason, none of the llvm.coro intrinsics are listed there.